indra/llwindow/llwindowsdl.h

```diff
@@ -46,83 +46,139 @@
+
+    bool isClipboardTextAvailable() override;
+
+    bool pasteTextFromClipboard(LLWString &dst) override;
+
+    bool copyTextToClipboard(const LLWString &src) override;
+
+    bool isPrimaryTextAvailable() override;
+
+    bool pasteTextFromPrimary(LLWString &dst) override;
+
+    bool copyTextToPrimary(const LLWString &src) override;
+
+    void flashIcon(F32 seconds) override;
+
+    F32 getGamma() override;
+
+    bool setGamma(const F32 gamma) override; // Set the gamma
+    U32 getFSAASamples() override;
+
+    void setFSAASamples(const U32 samples) override;
+
+    bool restoreGamma() override;            // Restore original gamma table (before updating gamma)
+    ESwapMethod getSwapMethod()  override { return mSwapMethod; }
+
+    void processMiscNativeEvents() override;
+
+    void gatherInput() override;
+
+    void swapBuffers() override;
+
+    void restoreGLContext() {};
+
+    void delayInputProcessing()  override {};
 
     // handy coordinate space conversion routines
-    /*virtual*/ bool convertCoords(LLCoordScreen from, LLCoordWindow *to);
-    /*virtual*/ bool convertCoords(LLCoordWindow from, LLCoordScreen *to);
-    /*virtual*/ bool convertCoords(LLCoordWindow from, LLCoordGL *to);
-    /*virtual*/ bool convertCoords(LLCoordGL from, LLCoordWindow *to);
-    /*virtual*/ bool convertCoords(LLCoordScreen from, LLCoordGL *to);
-    /*virtual*/ bool convertCoords(LLCoordGL from, LLCoordScreen *to);
+    bool convertCoords(LLCoordScreen from, LLCoordWindow *to) override;
+
+    bool convertCoords(LLCoordWindow from, LLCoordScreen *to) override;
+
+    bool convertCoords(LLCoordWindow from, LLCoordGL *to) override;
+
+    bool convertCoords(LLCoordGL from, LLCoordWindow *to) override;
+
+    bool convertCoords(LLCoordScreen from, LLCoordGL *to) override;
 
-    /*virtual*/ LLWindowResolution* getSupportedResolutions(S32 &num_resolutions);
-    /*virtual*/ F32 getNativeAspectRatio();
-    /*virtual*/ F32 getPixelAspectRatio();
-    /*virtual*/ void setNativeAspectRatio(F32 ratio) { mOverrideAspectRatio = ratio; }
+    bool convertCoords(LLCoordGL from, LLCoordScreen *to) override;
 
-    /*virtual*/ void beforeDialog();
-    /*virtual*/ void afterDialog();
+    LLWindowResolution *getSupportedResolutions(S32 &num_resolutions) override;
 
-    /*virtual*/ bool dialogColorPicker(F32 *r, F32 *g, F32 *b);
+    F32 getNativeAspectRatio() override;
 
-    /*virtual*/ void *getPlatformWindow();
-    /*virtual*/ void bringToFront();
+    F32 getPixelAspectRatio() override;
 
-    /*virtual*/ void spawnWebBrowser(const std::string& escaped_url, bool async);
+    void setNativeAspectRatio(F32 ratio)  override { mOverrideAspectRatio = ratio; }
+
+    void beforeDialog() override;
+
+    void afterDialog() override;
+
+    bool dialogColorPicker(F32 *r, F32 *g, F32 *b) override;
+
+    void *getPlatformWindow() override;
+
+    void bringToFront() override;
+    
```

Comment:
```suggestion
    void bringToFront() override;

```
For pre-commit to pass the check.